sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.RestClient.cs

```diff
@@ -91,46 +89,37 @@ private void OnPost()
 
                 var dataPoint = GetDataPoint();
 
-                var response = _quickPulseSDKClientAPIsRestClient.PostCustom(
+                var response = _quickPulseSDKClientAPIsRestClient.CustomPost(
                     ikey: _connectionVars.InstrumentationKey,
                     apikey: null,
                     xMsQpsConfigurationEtag: _etag,
                     xMsQpsTransmissionTime: null,
                     monitoringDataPoints: new MonitoringDataPoint[] { dataPoint }, // TODO: CHECK WITH SERVICE TEAM. WHY DOES THIS NEED TO BE A COLLECITON?
                     cancellationToken: default);
 
-                if (IsResponseSuccess(response))
+                if (response.Success)
                 {
-                    _lastSuccessfulPost = DateTimeOffset.UtcNow;
+                    _lastSuccessfulPing = DateTimeOffset.UtcNow;
```

Comment:
_lastSuccessfulPing - Ping or Post?

Reply:
copy paste error. This should be Post, thank you for catching that.